src/js/bellows.js

```diff
@@ -176,7 +176,7 @@
         close: function($item) {
             $item = this._item($item);
 
-            if (!$item.hasClass(cssClasses.OPENED)) {
+            if (!$item.hasClass(cssClasses.OPENED) || $item.hasClass(cssClasses.DISABLED)) {
```

Comment:
I see your reasoning @scalvert but I don't really understand why we should encapsulate here, since a single function such as `_isOpen` won't adequately capture the logic. If we would have lumped in the disabled check with the opened check then negating both of them would be the wrong logic.